cmd/boulder-sa/main.go

```diff
@@ -55,6 +55,7 @@ func main() {
 	dbMap, err := sa.NewDbMap(dbURL, saConf.DBConfig.MaxDBConns)
 	cmd.FailOnError(err, "Couldn't connect to SA database")
 
+	dbMap.Db.SetMaxIdleConns(saConf.DBConfig.MaxIdleDBConns)
```

Comment:
I think `test/config-next/sa.json` needs to have a `MaxIdleDBConns` value set. Similarly, since it will default to `0` maybe that should be detected and changed to `2` to match the Go default? I think as-is this would cause problems when the SA is upgraded but the config hasn't been switched over since 0 idle db conns will be allowed.

Reply:
Good catch!